### PR TITLE
Prepare for Docker publishing

### PR DESCRIPTION
**Pull request by kbariotis**
* set the final Docker namespace
* setup docker-compose for local development only
* add instructions and more information about the Docker image

Also addresses:
- https://github.com/excalidraw/excalidraw/issues/1768
- https://github.com/excalidraw/excalidraw/issues/1764

### CODE REVIEW

**Comment by kbariotis** [comment]
Waiting final confirmation from Docker and this will be ready to go.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,27 @@
-version: "3"
+version: "3.4"
```
Comment (mickeypash) [inline docker-compose.yml:1]: Hi, I was wondering why you're pinning to version `3.4` when the latest version is `3.8` is there a particular incompatibility?

Comment (kbariotis) [reply docker-compose.yml:1]: Hey there, no not all, happened to have a snippet of an example docker-compose file that seems to be outdated. :P 

I can try and update.

Comment (mickeypash) [reply docker-compose.yml]: Makes sense 👍

**Comment by kbariotis** [comment]
This is ready to go. https://hub.docker.com/r/excalidraw/excalidraw

**Comment by dwelle** [comment]
LGTM but I'm not a Docker guy.

Didn't know CRA supports the `<% %>` template logic, so that's cool.

**Pull request merged**